src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -200,7 +195,11 @@ EPOCHS = 2; // make test faster in Debug
 
       SDR goldSP({COLS});
       const SDR_sparse_t deterministicSP{
+#ifdef NTA_LIBC_MUSL
+        62, 72, 73, 82, 85, 102, 263, 277, 287, 303, 306, 308, 309, 322, 337, 339, 340, 352, 370, 493, 1094, 1095, 1114, 1115, 1120, 1463, 1512, 1518, 1647, 1651, 1691, 1694, 1729, 1745, 1746, 1760, 1770, 1774, 1775, 1781, 1797, 1798, 1803, 1804, 1805, 1812, 1827, 1828, 1831, 1832, 1858, 1859, 1860, 1861, 1862, 1875, 1878, 1880, 1881, 1898, 1918, 1923, 1929, 1931, 1936, 1950, 1953, 1956, 1958, 1961, 1964, 1965, 1967, 1971, 1973, 1975, 1976, 1979, 1980, 1981, 1982, 1984, 1985, 1986, 1988, 1991, 1994, 1996, 1997, 1998, 1999, 2002, 2006, 2008, 2011, 2012, 2013, 2017, 2019, 2022, 2027, 2030
```

Comment:
added custom deterministic results for MUSL. I don't know if we can achieve same on glibc & musl.

Reply:
See #707